### DISCUSSION TITLE
Go to parent note

### DISCUSSION THREAD

**Discussion by dmych** [SilentVoid13/Templater#992] [Templates Showcase]
Using [Folder Notes](https://github.com/aidenlx/alx-folder-note) plugin, I found no fast way to go to the parent note, so I made [this template](https://gist.github.com/dmych/45dbb4e822b67b9021f817d00706cf9f) which I assigned to `Cmd/Ctrl+U`:

```javascript
<%*    
let parName = tp.file.folder(true) + '.md';
console.log('File to open: "'+parName+'"');
if (await tp.file.exists(parName)) {
    app.workspace.openLinkText('', parName);
}
%>
```

**Comment by LuiZzz-Menezes** [comment]
I thought this would work for notes that are in "MOCs". For example, I have a note inside a note, I thought the template would return to "Parental Note", that is, the previous note

**Reply by dmych** [reply]
What do you mean "note inside a note"? In my system, there are no notes inside, but all notes located in a folder are child notes to that folder, and the folder note is a parent note to them.
This template dedicated to work with Folder Notes plugin, so it works when you have the note hierarchy based on folders:
```
Vault
   |-- Note.md
   |-- Folder ( + Folder.md file as a folder note)
   |         |-- Child1.md
   |         |-- Child2.md
   |-- Another folder
   ...
```
Here `/Folder.md` is a parent note for `Child1.md` and `Child2.md`. And when in any of these notes, the template will open `Folder.md` file.
